xtas/tests/test_frog.py

```diff
@@ -94,4 +94,4 @@ def test_frog_task():
     assert_equal(tokens[0]['lemma'], 'dit')
     saf = frog("dit is een test", output='saf')
     assert_equal(len(saf['tokens']), 4)
-    assert_equal(saf['header']['processed'][0]['module'], 'frog')
```

Comment:
Should we spot-check the output as well, like for the other output formats, to be a bit more sure that it worked as intended?